sklearnex/basic_statistics/basic_statistics.py

```diff
@@ -42,36 +42,61 @@ class BasicStatistics(BaseEstimator):
     """
     Estimator for basic statistics.
     Allows to compute basic statistics for provided data.
-    Note, some results can exhibit small variations due to
-    floating point error accumulation and multithreading.
 
     Parameters
     ----------
     result_options: string or list, default='all'
-        List of statistics to compute
+        Used to set statistics to calculate. Possible values are ``'min'``, ``'max'``, ``'sum'``, ``'mean'``, ``'variance'``,
+        ``'variation'``, ``sum_squares'``, ``sum_squares_centered'``, ``'standard_deviation'``, ``'second_order_raw_moment'``
+        or a list containing any of these values. If set to ``'all'`` then all possible statistics will be 
+        calculated.
 
-    Attributes (are existing only if corresponding result option exists)
+    Attributes 
     ----------
-        min : ndarray of shape (n_features,)
+        min_ : ndarray of shape (n_features,)
             Minimum of each feature over all samples.
-        max : ndarray of shape (n_features,)
+        max_ : ndarray of shape (n_features,)
             Maximum of each feature over all samples.
-        sum : ndarray of shape (n_features,)
+        sum_ : ndarray of shape (n_features,)
             Sum of each feature over all samples.
-        mean : ndarray of shape (n_features,)
+        mean_ : ndarray of shape (n_features,)
             Mean of each feature over all samples.
-        variance : ndarray of shape (n_features,)
+        variance_ : ndarray of shape (n_features,)
             Variance of each feature over all samples.
-        variation : ndarray of shape (n_features,)
+        variation_ : ndarray of shape (n_features,)
             Variation of each feature over all samples.
-        sum_squares : ndarray of shape (n_features,)
+        sum_squares_ : ndarray of shape (n_features,)
             Sum of squares for each feature over all samples.
-        standard_deviation : ndarray of shape (n_features,)
+        standard_deviation_ : ndarray of shape (n_features,)
             Standard deviation of each feature over all samples.
-        sum_squares_centered : ndarray of shape (n_features,)
+        sum_squares_centered_ : ndarray of shape (n_features,)
             Centered sum of squares for each feature over all samples.
-        second_order_raw_moment : ndarray of shape (n_features,)
+        second_order_raw_moment_ : ndarray of shape (n_features,)
             Second order moment of each feature over all samples.
+
+    Note
+    ----
+        Attribute exists only if corresponding result option has been provided.
+    Note
+    ----
+        Attributes' names without the trailing underscore are
+        supported currently but deprecated in 2025.1 and will be removed in 2026.0
+    Note
+    ---- 
+    Some results can exhibit small variations due to
+    floating point error accumulation and multithreading.
```

Comment:
It depends. I wanted to highlight it is different notes, not related to each other.

Reply:
I would suggest to add Limitation instead of Note, based on the your notes here.
And again for me it is better to add the section definition at once. Needed to check guidelines

Reply:
@samir-nasibli It's unfortunately not possible to add sections with arbitrary titles. "Note" is supported by the numpy docstring specification, but "limitation" isn't:
https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
> And again for me it is better to add the section definition at once

It affects the rendering. The section defined once would look differently.